src/GapicClientTrait.php

```diff
@@ -762,6 +830,67 @@ private function buildMethod($interfaceName, $methodName)
         );
     }
 
+    /**
+     * @param array $headerParams
+     * @param Message $request
+     *
+     * @return array
+     *
+     * @internal
+     */
+    public function buildRequestParamsHeader($headerParams, $request)
```

Comment:
> If it's solely for testability, we can add a wrapper method in the GapicClientTraitStub class for testing it.

It is solely for testability. Can you point at an example of said "wrapper method"?

Reply:
Yes! [This stub](https://github.com/googleapis/gax-php/blob/main/tests/Tests/Unit/GapicClientTraitTest.php#L1246) would allow you to do:

```php
$stub = new GapicClientTraitStub();
$response = $stub->call('buildRequestParamsHeader', [$headerParams, $request]);
```

Reply:
Awesome! Done!